clar/sandbox.h

```diff
@@ -2,8 +2,17 @@
 #include <sys/syslimits.h>
 #endif
 
+/*
+ * The tempdir is the temporary directory for the entirety of the clar
+ * process execution. The sandbox is an individual temporary directory
+ * for the execution of an individual test. Sandboxes are deleted
+ * entirely after test execution to avoid pollution across tests.
+ */
 #define CLAR_PATH_MAX 4096
-static char _clar_path[CLAR_PATH_MAX];
+static char _clar_tempdir[CLAR_PATH_MAX];
+size_t _clar_tempdir_len;
```

Comment:
Does this need to be non-static? I couldn't spot any uses of it outside of the current scope.